### ISSUE TITLE
Paver is not centering on mobiles

### ISSUE THREAD

**Issue by PabloArty** [terrymun/paver#14]
Morning! I'm having this problem on phones, the paver image just goes all to the left and I want it to be centered. Is ther any option for this?

**Comment by terrymun** [comment]
That is very strange: on mobile the Paver image should be centered no matter what. Does your mobile device provide gyroscopic data? There is a possibility that paver is falling back to the default `overflow: scroll` trick if no gyroscope data is available.

**Comment by PabloArty** [comment]
Yeah is an Iphone it has gyroscope. D: I saw some parts of paver js that have something about the gyroscope so i will check them if I found something I'll put it here, but if you know what could be the problem please tell me ;)
